egs/sre08/v1/sid/nnet3/xvector/get_egs_seg.sh

```diff
@@ -0,0 +1,321 @@
+#!/bin/bash
+
+stage=0
+cmd=run.pl
+nj=60
+data=data/swbd_sre_combined_no_sil
+dir=exp/xvector_nnet/egs
+
+#some options
+num_heldout_utts=1000       # number of utterances held out for training subset
+                            # and validation set
+min_frames_per_chunk=200
+max_frames_per_chunk=400
+
+kinds_of_length_train=100   # number of the kinds of lengths in train set
+kinds_of_length_valid=3     # number of the kinds of lengths in valiation set
+                            # and training subset
+
+frames_per_iter=70000000    # target number of frames per archive. If it is null,
```

Comment:
Is n't it too large for frames_per_iter? Did you try smaller number?

Reply:
I just try to match david's setup. In his setup, each archive contains 179865 entries. 179865*400~= 70million frames.
I will open a new pull request. Let's deal with your case there.